turan.tex

```diff
@@ -33,9 +33,14 @@
 
 \title{\MakeUppercase{More Turán-Type Theorems for Triangles in Convex Point Sets}\thanks{This research is partially funded by NSERC.}}
 
-\author{Boris Aronov, Vida Dujmović, Ruy Fabila, Pat Morin, Aurélien Ooms,\\
-and Luís Fernando Schultz Xavier da Silveira  }
-
+\author{%
+Boris Aronov,
+Vida Dujmović,
+Ruy Fabila,
+Pat Morin,
+Aurélien Ooms\thanks{Supported by the Fund for Research Training in Industry and Agriculture (FRIA).},\\
```

Comment:
I should just delete turan.py. It's not used anymore and it's definitely out of date.  turan-table.py generates the two tables.